### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by xiaolinggnb**
Added information about the scaling effect, so that the description of the effect after using “contain” is more accurate.

### Description

See Changes section for details, only description text added.

### CODE REVIEW

File: files/en-us/web/css/object-fit/index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ The `object-fit` property is specified as a single keyword chosen from the list
 ### Values
 
 - `contain`
-  - : The replaced content is scaled to maintain its {{glossary("aspect ratio")}} while fitting within the element's content box. The entire object is made to fill the box, while preserving its aspect ratio, so the object will be ["letterboxed"](<https://en.wikipedia.org/wiki/Letterboxing_(filming)>) if its aspect ratio does not match the aspect ratio of the box.
+  - : The replaced content is scaled to maintain its {{glossary("aspect ratio")}} while fitting within the element's content box. The entire object is made to fill the box, while preserving its aspect ratio, so the object will be ["letterboxed"](<https://en.wikipedia.org/wiki/Letterboxing_(filming)>) or ["pillarboxed"](<https://en.wikipedia.org/wiki/Pillarbox>) if its aspect ratio does not match the aspect ratio of the box.
```
[the remark this answers is not in the document]
Comment (xiaolinggnb) [reply files/en-us/web/css/object-fit/index.md:41]: I don't understand whether `<>` signs should be removed or not.

[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/object-fit/index.md:41]: I think the `<>` are needed if there are parenthesis in the URL, but not otherwise.

**Review by estelle — APPROVED** [review]
Thank you. Also, congratulations on your first merged MDN content PR. Welcome to he team!

File: files/en-us/web/css/object-fit/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (xiaolinggnb) [reply files/en-us/web/css/object-fit/index.md]: Thanks for explaining—it all makes sense now!

**Comment by xiaolinggnb** [comment]
It’s my pleasure, and thank you for all your hard work as well!

**Pull request merged**